src/components/Identifier/IdentifierMasthead/IdentifierMasthead.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactElement } from 'react'
+import classnames from 'classnames'
+
+import { IdentifierLogo } from '../IdentifierLogo/IdentifierLogo'
+
+type Language = 'english' | 'spanish'
+
+interface IdentifierMastheadCopyMap {
+  ariaLabelAgencyIdentifier: string
+  ariaLabelAgencyDescription: string
+  identityDisclaimer: string
+}
+
+const copyMap: Record<Language, IdentifierMastheadCopyMap> = {
+  english: {
+    ariaLabelAgencyIdentifier: 'Agency identifier',
+    ariaLabelAgencyDescription: 'Agency description',
+    identityDisclaimer: 'An official website of the',
+  },
+  spanish: {
+    ariaLabelAgencyIdentifier: 'Identificador de la agencia',
+    ariaLabelAgencyDescription: 'Descripción de la agencia',
+    identityDisclaimer: 'Un sitio web oficial de',
+  },
+}
+
+interface ParentAgency {
+  url: string
+  name: string
+  logo?: string
+}
+
+interface IdentifierMastheadProps {
+  parentAgency: ParentAgency
+  domain: string
+  language?: Language
+  taxpayerDisclaimer?: string
+  className?: string
+}
+
+export const IdentifierMasthead = ({
+  parentAgency,
+  domain,
+  language = 'english',
+  taxpayerDisclaimer,
+  className,
+  ...sectionProps
+}: IdentifierMastheadProps &
+  JSX.IntrinsicElements['section']): React.ReactElement => {
+  const classes = classnames(
+    'usa-identifier__section usa-identifier__section--masthead',
+    className
+  )
+
+  const copy = copyMap[`${language}` as Language]
+
+  return (
+    <section
+      data-testid="identifierMasthead"
+      className={classes}
+      aria-label={copy.ariaLabelAgencyIdentifier}
+      {...sectionProps}>
+      <div className="usa-identifier__container">
+        <div className="usa-identifier__logos">
+          {parentAgency.logo && (
+            <IdentifierLogo
+              agencyUrl={parentAgency.url}
+              agencyName={parentAgency.name}
+              src={parentAgency.logo}
+            />
+          )}
+          <div
+            data-testid="identifierMasthead-agency-description"
+            className="usa-identifier__identity"
+            aria-label={copy.ariaLabelAgencyDescription}>
+            <p className="usa-identifier__identity-domain">{domain}</p>
+            <p className="usa-identifier__identity-disclaimer">
+              {copy.identityDisclaimer}
+              &nbsp;
+              <a href={parentAgency.url}>{parentAgency.name}</a>
+              {taxpayerDisclaimer && `. ${taxpayerDisclaimer}`}
+            </p>
+          </div>
+        </div>
+      </div>
```

Comment:
I noticed that there isn't as much whitespace between the logo and the agency & domain text in the storybook screenshots as there is in the USWDS examples. I _think_ that's just because one of the closing `</div>` tags needs to be before the `identifierMasthead-agency-description` div. It's that sneaky one here in their source code:

![image](https://user-images.githubusercontent.com/15805554/112544041-f1681480-8d8c-11eb-89e4-547e1d0af514.png)

Reply:
Confirming that that fixed it, thank you for spotting that!

![Screen Shot 2021-03-26 at 11 10 05 AM](https://user-images.githubusercontent.com/16230705/112674999-d8f70900-8e23-11eb-81f8-fe4c5d176ca5.png)

Reply:
Nice! Looks great.